lib/services/consumers/application.service.js

```diff
@@ -51,11 +60,20 @@ s.find = function (appName) {
     });
 };
 
-s.findAll = function (query = {}) {
+s.findAll = function (query) {
+  const userService = require('./user.service');
```

Comment:
I'm not sure that this preload is required all the time. And what if one needs more info, like Firstname?  

how about `/users` will allow query params to filter by ids
something like `/users?ids=12,34,45`=> List of Users